src/ott/solvers/nn/neuraldual.py

```diff
@@ -327,3 +600,124 @@ def _penalize_weights_icnn(params: Dict[str, jnp.ndarray]) -> float:
       if k.startswith("w_z"):
         penalty += jnp.linalg.norm(jax.nn.relu(-param["kernel"]))
     return penalty
+
+
+def plot_ot_map(
+    learned_potentials: potentials.DualPotentials,
+    source: jnp.ndarray,
+    target: jnp.ndarray,
+    inverse: bool = False
+) -> Tuple[matplotlib.figure.Figure, matplotlib.axes.Axes]:
```

Comment:
Please add `matplotlib` to the [intersphinx_mapping](https://github.com/ott-jax/ott/blob/main/docs/conf.py#L68).

Reply:
added `"matplotlib": ('https://matplotlib.org/stable/', None)`